queries/idl/injections.scm

```diff
@@ -0,0 +1,3 @@
+((comment) @injection.content
```

Comment:
Normally you'd inject the `comment` parser here; you should then add an injection for `(annotation_comment)` in `idl_annotation`.

Reply:
what you means is this?
```scheme
((comment) @injection.content
  (#set! injection.language "comment"))

((comment) @injection.content
  (#lua-match? @injection.content "/[*\/][!*\/]<?[^a-zA-Z]")
  (#set! injection.language "doxygen"))

((comment) @injection.content
  (#lua-match? @injection.content "//@[a-zA-Z]")
  (#set! injection.language "idl_annotation"))
```

Reply:
so it should be this?

```scheme
((comment) @injection.content
  (#lua-match? @injection.content "/[*\/][!*\/]<?[^a-zA-Z]")
  (#set! injection.language "doxygen"))

((comment) @injection.content
  (#lua-match? @injection.content "//@[a-zA-Z]")
  (#set! injection.language "idl_annotation"))
```

These two injections will not be selected at the same time

Reply:
And a 
```
((comment) @injection.content
  (#not-lua-match? @injection.content "/[*\/][!*\/]<?[^a-zA-Z]")
  (#not-lua-match? @injection.content "//@[a-zA-Z]")
  (#set! injection.language "comment"))
```

Reply:
(Tbh comment parser is a special case, you can have multiple injections with comment nodes, but only comment nodes)

Reply:
No, I mean add injections for the injected `idl_annotation` language.

Reply:
> Normally you'd inject the `comment` parser here; you should then add an injection for `(annotation_comment)` in `idl_annotation`.

Sorry, maybe my naming caused confusion. In fact, idl_annotation is used to parse annotation_comment, so it should not need injection:

```js
module.exports = grammar({
  name: 'idl_annotation',
  rules: {
    specification: $ => repeat1($.annotation_comment),
  })
```